app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Dish</h1>
+
+<p>Name: <%= @dish.name %> </p>
+<p>Description: <%= @dish.description %> </p>
+<p>Ingredients: 
+  <%  @dish.ingredients.each do |ingredient| %>
+  <%= ingredient.name  %> <%= button_to "Remove #{ingredient.name}", "/chef/#{@chef.id}/dishes/#{@dish.id}/ingredients/#{ingredient.id}", method: :delete %> </b>
```

Comment:
Instead of having to find the chef in the controller and then passing that to the view I think you could have just done `@dish.chef_id` since every dish belongs to a chef.

```ruby
<%= button_to "Remove #{ingredient.name}", "/chef/#{@dish.chef_id}/dishes/#{@dish.id}/ingredients/#{ingredient.id}", method: :delete %>
```

Moving past that, however, I don't think you even need a chef to be able to remove an ingredient from a dish since all it would require you to do would be to remove that row from the join table.

```ruby
<%= button_to "Remove #{ingredient.name}", "/dishes/#{@dish.id}/ingredients/#{ingredient.id}", method: :delete %>
```